keras_unet/models/custom_unet.py

```diff
@@ -24,53 +25,91 @@ def conv2d_block(
     inputs,
     use_batch_norm=True,
     dropout=0.3,
+    dropout_type="spatial",
     filters=16,
     kernel_size=(3, 3),
     activation="relu",
     kernel_initializer="he_normal",
     padding="same",
 ):
 
+    if dropout_type == "spatial":
+        DO = SpatialDropout2D
+    elif dropout_type == "standard":
+        DO = Dropout
+    else:
+        raise ValueError(
+            f"dropout_type must be one of ['spatial', 'standard'], got {dropout_type}"
+        )
+
     c = Conv2D(
         filters,
         kernel_size,
-        activation=None,
+        activation="relu",
```

Comment:
Can you change it to `activation=activation,` so that we don't have it hardcoded?

Reply:
Yeah sorry, fixed it